assets/src/stories-editor/components/resizable-box/helpers.js

```diff
@@ -190,3 +191,46 @@ export const getBlockPositioning = ( width, height, radian, direction ) => {
 		top: rotatedY - y,
 	};
 };
+
+/**
+ * Get block's adjusted left and top position after it has been resized.
+ *
+ * @param {Object}   attributes Attributes.
+ * @return {Object}  Top and left positions adjusted based on the rotation angle.
+ */
+export const getPositionAfterResizing = ( attributes ) => {
+	const {
+		direction,
+		angle,
+		isText,
+		width,
+		height,
+		appliedWidth,
```

Comment:
I think it might be a bigger task to change the name of this variable as it used in the other component like this. Could we just add jsdocs explaining the attributes.

Reply:
Both seemed like good ideas, so did both: 
641908c